docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+
+Verzoeken zijn voor het eerst geintroduceerd in GFO-Zaken, maar is daarna weer verdwenen in het opvolgende informatiemodel RGBZ. In de visie van VNG Realisatie is dat echter een omissie, en is er behoefte aan een concept dat vooraf gaat aan de daadwerkelijke behandeling in een zaak (of andere activiteit).
+
+# Intake
+Op een verzoek wordt intake gedaan. Daarbij wordt het verzoek niet inhoudelijk beoordeeld, maar wordt wel gekeken of de  aangevraagde producten geleverd kunnen worden en of alle benodigde gegevens zijn ingevuld. Na de intake is het verzoek in behandeling genomen dan wel buiten behandeling gesteld. Een verzoek kan uitmonden in een of meerdere zaken of andere activiteiten (d.w.z. niet zaakgericht), dan wel direct worden afgehandeld, bijv. een product zoals een folder dat direct kan worden geleverd. Ook kan een verzoek worden gekoppeld aan reeds lopende zaak. Denk bijvoorbeeld aan een Melding Openbare Ruimte die reeds eerder is gedaan en waarvoor reeds een zaak loopt.
+
+Status van verzoeken: is in feite de status van de intake: nieuw, ingetrokken, in intake/behandeling, in behandeling genomen, afgewezen. Nadat een verzoek in behandeling is genomen (of afgewezen) kan het niet meer worden gewijzigd.
+
+# Interactie met de klant
+Terugkoppeling naar de klant over voortgang gebeurt in termen van de opgestarte zaken of andere activiteiten. Een verzoek heeft alleen een status m.b.t. voortgang van de intake die overigens ook in de PIP getoond kan worden (let op: dit kan verwarrend zijn als ook statussen van zaken getoond worden; dit is aan de consumers om op te lossen). De klant kan een overzicht van ingediende verzoeken opvragen en moet een verzoek kunnen intrekken. Een verzoek tot intrekken wordt gerelateerd aan het oorspronkelijke verzoek. Ook kan een klant aanvullingen doen op een verzoek. Dat leidt tot een nieuw verzoek dat aan het oorspronkelijke verzoek wordt gekoppeld.
+
+Een verzoek kan meerdere producten/diensten omvatten hetgeen een winkelmandje mogelijk maakt op de website van de gemeente. Als gevolg hiervan kan een verzoek tot meerdere zaken leiden. 
+
+# Gegevens: generiek en specifiek
+Bij een verzoek wordt alleen generieke metadata opgeslagen. Procesgegevens (domeinspecifiek) worden in een andere registratie opgeslagen. Bijv. bij een Melding Openbare Ruimte worden specifieke gegevens zoals locatie en meldinggegevens in een meldingen-registratie opgeslagen. Vanuit het verzoek wordt verwezen met een URI naar de melding.
+
+Wat betreft behandeltermijn (uiterste en geplande) volgen we de zaaktypen. Terzijde: deze zouden eigenlijk bij de producten in de PDC moeten staan, omdat producten de interface vormen tussen klant en dienstverlener. Voor nu volgen we het huidige model.
```

Comment:
Geef wat meer uitleg wat 

>Wat betreft behandeltermijn (uiterste en geplande) volgen we de zaaktypen

betekent.
Je zou zeggen dat de behandeltermijn van een verzoek wordt bepaald door behandeltermijn van die er aan gekoppelde zaak die het langste is.